data-science-pipelines-operator/base/params.env

```diff
@@ -7,3 +7,6 @@ IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
 IMAGES_DSPO=registry.redhat.io/rhods/odh-data-science-pipelines-operator-controller-rhel8:latest
 IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+IMAGES_MLMDENVOY=upstream-only
+IMAGES_MLMDGRPC=upstream-only
+IMAGES_MLMDWRITER=upstream-only
```

Comment:
This is a downstream specific change, to confirm this doesn't break here's an example of it running using the live builder image provided: 


![pr-413](https://github.com/red-hat-data-services/odh-manifests/assets/10904967/02fe4bc1-b658-4484-9939-8ad508fbe4df)